modules/loader/src/main/scala/com/snowplowanalytics/snowplow/rdbloader/config/CliConfig.scala

```diff
@@ -45,15 +39,18 @@ object CliConfig {
 
   val config = Opts.option[String]("config",
     "base64-encoded HOCON configuration", "c", "config.hocon")
+    .mapValidated(x => ConfigUtils.base64decode(x).flatMap(Config.fromString).toValidatedNel)
```

Comment:
It feels to me that it's slightly unfair to use plain `Either` here because we also rely on system properties and `application.conf` a lot. I mean we always could use them, but not it's much more important part and I believe it should be `EitherT`.

Perhaps it's fine to leave `fromConfig` itself working with `Either` because we use from batch shredder as well, but in Loader and Stream Shredder I'd make it `EitherT`.